### PR TITLE
Consolidate Monitoring section: runs pages (23.3 and 23.4)

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Per https://github.com/seqeralabs/nf-tower-docs/issues/655

Collapse existing Monitoring section into 3 pages: Runs overview, Dashboard, and Run details. 

These 3 pages are in a Runs subsection in Monitoring, to prepare the nav section for [Cloud costs](https://github.com/seqeralabs/docs/pull/19/files) and [Audit logs](https://github.com/seqeralabs/docs/pull/17) pages.

### CODE REVIEW

File: platform_versioned_docs/version-23.3.0/monitoring/dashboard.mdx [hunk 1]
```diff
@@ -23,7 +19,7 @@ Select **Export data** in the filter panel near the top of the page to export da
 
 ### Runs per organization
 
-For each organization you have access to, run totals for your selected filters are displayed.
+Run totals for your selected filters are displayed for each organization you have access to.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/monitoring/dashboard.mdx:22]:
```suggestion
Run totals for your selected filters are displayed for each organization that you have access to.
```

File: platform_versioned_docs/version-23.3.0/monitoring/overview.mdx [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+You can **Review and edit** any run details prior to saving the pipeline. After you've saved the pipeline, it is listed on the **Launchpad** and can be run from the same workspace where it was created.
 
-### All runs view
+## All runs view
 
-_Available from version 22.4.0_
+The **All runs** page, accessed from the user menu, provides a comprehensive overview of the runs accessible to a user across the entire Seqera instance. This facilitates overall status monitoring and early detection of execution issues from a single view, split across organizations and workspaces.
 
-The **All runs** page, accessed from the user menu, provides a comprehensive overview of the runs accessible to a user across the entire Seqera instance. This facilitates overall status monitoring and early detection of execution issues from a single view split across organizations and workspaces.
+The All runs view defaults to all organizations and workspaces you can access. Select the dropdown next to **View** to filter by specific organizations and workspaces, or to view runs from your personal workspace only.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/monitoring/overview.mdx:33]: If _All runs_ is a specific UI area, section, or element, in this case do we not want to use bold for it?

File: platform_versioned_docs/version-23.3.0/monitoring/overview.mdx [hunk 3]
```diff
@@ -47,63 +47,50 @@ The **Search workflow** bar allows you to filter by one or more `<keyword>:<valu
...
+- **Freeform text**
 
-- `after:2022-04-06`: search workflows submitted on or after the given date in YYYY-MM-DD format.
+  The search box allows you to search for partial matches with `project name`, `run name`, `session id`, or `manifest name`. Use wildcards (`*`) before or after keywords to filter results.
 
-- `label:label1 label:label2`: search workflows with specific labels.
+- **Exact match keywords**
 
-- `is:starred`: search workflows that have been starred by the user.
+  - `workflowId:3b7ToXeH9GvESr`: search workflows with a specific workflow ID.
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/monitoring/overview.mdx:87]: I wonder about capitalizing _s_ in this context?

File: platform_versioned_docs/version-23.3.0/monitoring/run-details.mdx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+### Run information
+
+This section contains details about the Nextflow execution:
+
+- The Nextflow **Command line** executed.
+- The pipeline **Parameters** (taken from the configuration `params` scope).
+- The **Configuration** files and the final resolved configuration.
+- The **Execution log** from the main Nextflow process, updated in real-time.
+- Available **Reports**, if any are [configured](../reports/overview.mdx).
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.3.0/monitoring/run-details.mdx:24]:
```suggestion
- Available **Reports**, if any, are [configured](../reports/overview.mdx).
```

**Review by jason-seqera — APPROVED** [review]
Left a few minor comments, thanks!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**